content/docs/for-app-developers/concepts/base-images/stack.md

```diff
@@ -42,8 +42,8 @@ Stacks are used by [builders][builder] and are configured through a builder's
 
 [stack]
   id = "com.example.stack"
-  build-image = "example/build"
-  run-image = "example/run"
+  build-image = "docker.io/example/build"
```

Comment:
Does this image need a registry?  I think this may meet your criteria of only being used in the local environment

Reply:
In my understanding, this image means it is pulled from a remote registry to be used as a build environment, so I thought it should be specified with the registry name.

Reply:
This should be okay, the daemon will happily accept `some-image` and `index.docker.io/some-image` interchangeably.

Side note, while `docker.io/some-image` is correct, all of our code will resolve a registry-less image name to `index.docker.io/<image>` so I think using the fully qualified domain would be clearer.